radix-engine-tests/tests/bootstrap.rs

```diff
@@ -94,6 +94,85 @@ fn test_bootstrap_receipt_should_match_constants() {
     assert_eq!(wrap_up_epoch_change.epoch, genesis_epoch.next());
 }
 
+#[test]
+fn test_bootstrap_receipts_should_have_complete_system_structure() {
```

Comment:
Nice!

You've written some useful code here, so I wonder if we should move it out to a separate `system_structure.rs` test file, like we've done with the `typed_substate_layout.rs` tests - so we can also use this helper code to validate (eg) the scenario receipts.

Reply:
Yeah, but it seems like a lower-prio refactor/improvement for later.
Let me add a TODO here and merge after CI's 🟢 .